watertap/unit_models/crystallizer.py

```diff
@@ -446,6 +473,14 @@ def eq_p_con2(b):
         def eq_p_con3(b):
             return b.properties_vapor[0].pressure == self.pressure_operating
 
+        @self.Constraint()
+        def eq_p_con4(b):
+            return b.properties_pure_water[0].pressure == self.pressure_operating
+
+        @self.Constraint()
+        def eq_p_con5(b):
+            return b.properties_pure_water[0].pressure_sat == self.pressure_operating
+
         @self.Constraint()
         def eq_T_con1(b):
```

Comment:
Though these constraints are nearly self-explanatory, they should have doc strings describing what they are (e.g., "Isobaric pressure constraint for pressure of vapor at crystallizer outlet"). I know a couple of these were already missing doc strings before this PR, so cleaning those up would be nice.